src/includes/enriching-events/attach-screenshots/unity.mdx

```diff
@@ -0,0 +1,13 @@
+The **Attach Screenshot** options can be found on the `Enrichment` tab in the Editor Configuration Window.
+
+![Screenshots Thumbnail Example](unity-screenshots.png)
+
+<Note>
+
+Note that capturing a screenshot in the Unity SDK requires running on the main thread.
```

Comment:
This is important to be written in the common docs as well (src/platforms/common/enriching-events/screenshots.mdx)